services/i18n-service/src/lib/translation-provider.ts

```
@@ -0,0 +1,202 @@
+  ): Promise<TranslationProvider> {
+    const { defaultProvider, googleApiKey } = config;
+
+    switch (defaultProvider) {
+      case 'google':
+        if (!googleApiKey) {
+          console.warn(
+            'Google Translation API key not configured, falling back to mock'
+          );
+          return new MockTranslationProvider();
+        }
+        const googleProvider = new GoogleTranslationProvider(googleApiKey);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)